Sources/Purchasing/TrialOrIntroPriceEligibilityChecker.swift

```diff
@@ -54,6 +54,14 @@ class TrialOrIntroPriceEligibilityChecker: TrialOrIntroPriceEligibilityCheckerTy
 
     func checkEligibility(productIdentifiers: Set<String>,
                           completion: @escaping ReceiveIntroEligibilityBlock) {
+        guard !self.systemInfo.dangerousSettings.uiPreviewMode else {
+            let result = productIdentifiers.reduce(into: [:]) { resultDict, productId in
+                resultDict[productId] = IntroEligibility(eligibilityStatus: IntroEligibilityStatus.unknown)
```

Comment:
Could be worth adding a comment... but NABD

Reply:
Done!